taichi/transforms/type_check.cpp

```diff
@@ -397,13 +397,20 @@ class TypeCheck : public IRVisitor {
   }
 
   void visit(SNodeLookupStmt *stmt) {
+    // TODO: for bit_array SNodes, an SNodeLookupStmt should convert a byte
+    // pointer to bit pointer.
+    // See the GetChStmt treatment below.
```

Comment:
@TH3CHARLie A TODO for you :-)

Reply:
Got it! Sorry I am busy in Hangzhou recently, will get back to work on Sunday.